### PR TITLE
[4.x] May 2018 survey banner

### PR DESCRIPTION
**Pull request by chalin**
4.x contribution to #1598 

Staged at https://webdev-dartlang-org-staging-0.firebaseapp.com

---

Sample screenshots are shown below. For pages other than the front page, I've tried a variant where there is no space between the two banners. Let me know which style you prefer (this style or the one illustrated for the v5 site).

> screen shot 2018-05-18 at 09 37 46

> screen shot 2018-05-18 at 09 37 51

### CODE REVIEW

File: src/_includes/banner-top.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<div class="banner alert alert-warning" markdown="1">
+  <i class="material-icons">web</i>
+  Do you build web apps with Dart?
+  Please take our
+  [survey][]{:.no-automatic-external target="_blank" rel="noopener"}!
+</div>
+
+[survey]: https://t.co/f16aEsDI3V
```
Comment (kevmoo) [inline src/_includes/banner-top.md]: Why using the twitter redirect?

Comment (chalin) [reply src/_includes/banner-top.md]: Ok, I'll change the URL to https://services.google.com/fb/forms/dart4web-may2018/

Comment (chalin) [reply src/_includes/banner-top.md]: Approved otherwise?

**Comment by kevmoo** [comment]
Yup!

**Pull request merged**